src/containers/Ratings/RatingArticle.js

```diff
@@ -42,7 +42,7 @@ export class Rating extends Component {
           total={5}
           onRate={this.handleRate.bind(this)}
         />{" "}
-        <span className="score center-align">
+        <span className="score center-align" data-test="ratings">
           {ratings ? ratings.toFixed(1) : 0}
```

Comment:
Awesome work @bencyn I just had a question on what `ratings.toFixed()` method does

Reply:
Good question @Tevinthuku  toFixed(1) converts an integer value to float with 1 precision that the decimal point so 4.toFixed(1) would result in 4.0. So the parameter indicates the number of decimal points. Have a look at the following resource for more insight https://www.w3schools.com/jsref/jsref_tofixed.asp